### ISSUE TITLE
Full documentation needed

### ISSUE THREAD

**Issue by pablo-clueless** [CommunityPro/ecommerce-api#35]
- Add all specs, code blocks and examples.

**Comment by nikomancy** [comment]
I'm interested in helping with this.

For the scope of the work:

- **Carts**
  - [ ] Add Item to Cart
  - [ ] Remove Item fromCart
  - [ ] Get Cart
  - [ ] Empty Cart
- **Customers**
  - [ ] Fetch Customer
  - [ ] Fetch Customers

Is that correct?

**Comment by pablo-clueless** [comment]
Yes. But we might also need to add screenshots of a curl request and response and a little bit of notes to the examples. Let me know if you're able to do that.

**Comment by pablo-clueless** [comment]
Yes. But we might also need to add screenshots of a curl request and response and a little bit of notes to the examples. Let me know if you're able to do that.

**Comment by nikomancy** [comment]
I can write up some notes given a general explanation of the resources and their use cases. I should be able to get a curl response and request set up. Also, what do you think of me rewriting the "sending your first request" section to have explicit instructions around how to do the authentication workflow? Since that is a gate on actually sending the first, it would probably be good to include it there.
